libraries/rib-compose/src/main/java/com/badoo/ribs/compose/ComposeTarget.kt

```diff
@@ -0,0 +1,41 @@
+package com.badoo.ribs.compose
+
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.viewinterop.AndroidView
+
+class ComposeTarget(
+    composeViewState: MutableState<ComposeView?>,
+    val container: ViewGroup,
+) {
+    constructor(
+        composeViewState: MutableState<ComposeView?>,
+        context: Context,
+    ) : this(
+        composeViewState = composeViewState,
+        container = FrameLayout(context).apply {
```

Comment:
This FrameLayout is hosted inside `androidx.ComposeView` which is not `FrameLayout`, so I do not think these params works. Can you check how it works without them?

Reply:
I did actually add it because it was not working fine. I will take some screenshots

Reply:
![Screenshot_20210921-160732](https://user-images.githubusercontent.com/12585526/134186703-e594d45b-b886-4960-9d21-86eeb12e20fb.png
![Screenshot_20210921-160710](https://user-images.githubusercontent.com/12585526/134186741-e1c16c21-3093-4241-8e98-850e0ec4baeb.png)
)